src/business_logic/execution/execution_errors.rs

```diff
@@ -0,0 +1,14 @@
+use thiserror::Error;
+
+use crate::core::errors;
+#[derive(Debug, PartialEq, Eq, Error)]
+pub enum ExecutionError {
+    #[error("Missing field for TxStruct")]
+    MissingTxStructField,
+    #[error("Expected an int value but get wrong data type")]
+    NotABigIntValue,
+    #[error("Expected a relocatable value but get wrong data type")]
+    NotARelocatableValue,
+    #[error("Error trying to cast data to other type")]
+    ErrorInDataConversion,
```

Comment:
Maybe we could add more info related to the conversion, something like:
```suggestion
    #[error("Error converting {0} into {1}")]
    ErrorInDataConversion(String, String),
```

Reply:
good one, Fixed